packages/versioning/test/incompatible-versioning.test.ts

```diff
@@ -227,6 +227,29 @@ describe("versioning: validate incompatible references", () => {
       });
     });
 
+    it("emit diagnostic when model property type using @typeChangedFrom does not yet exist", async () => {
```

Comment:
should there be a test the other way where `Original` didn't exists before v2 either

Reply:
Added